esss_environment.devenv.yml

```diff
@@ -45,10 +45,12 @@ dependencies:
     - fix-and-check-env>=1.0.0
     - invoke>=1.0.0
     - jinja2==2.10
-    - pyqt>=5.6,<5.7
+    - pyqt==5.12.3
```

Comment:
As a library, qmxgraph should not pin exactly. I suggest using `pyqt>=5.12` here.

Reply:
OK.
But note this is the `esss_environment.devenv.yml`